Documents/OrderingRules.md

```diff
@@ -0,0 +1,131 @@
+# Ordering Rules
+
+## The Abstract
+
+Rules which enforce a standard ordering scheme for code contents.
+
+## Rules
+
+Within a class, struct or interface
+
+* Constant Fields
+* Fields
+* Constructors
+* Finalizers (Destructors)
+* Delegates
+* Enums
```

Comment:
I prefer to have every Enum type in its own file. I think we should not include more that one Class or Enum in a file.